src/gui/transferlistmodel.cpp

```diff
@@ -205,8 +205,8 @@ QVariant TransferListModel::headerData(int section, Qt::Orientation orientation,
 
 QString TransferListModel::displayValue(const BitTorrent::TorrentHandle *torrent, const int column) const
 {
-    const bool isHideState = (Preferences::instance()->getHideZeroComboValues() == 1)
-            && (torrent->state() == BitTorrent::TorrentState::PausedDownloading); // paused torrents only
+    const bool isHideState = (Preferences::instance()->getHideZeroComboValues() != 1)
```

Comment:
I think it is a bit clearer to specify `Preferences::instance()->getHideZeroComboValues() == 0` for the first part which can only match the "Always" option in case we add more options later...

Reply:
OK.
PR is updated. I made another improvement on top of it.

OFFTOPIC.
Where were my eyes when we reviewed "hide zero values" feature? Why is this incorrect option name here? In such cases we don't store "state" of some controls! On the contrary, the control provides some option value.

BTW, we don't really need two options (and controls) to provide "hide zero values" feature. Only one "Hide zero and infinity values:" that can be assigned to "Never", "Paused" or "Always" (like I've made internally in TransferListModel in the second commit here).